products/memcached.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Memcached
+category: db
+iconSlug: memcached
+permalink: /memcached
+versionCommand: memcached -h
+activeSupportColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:generic/memcached
+-   purl: pkg:deb/ubuntu/memcached
+-   purl: pkg:deb/debian/memcached
+-   purl: pkg:rpm/amzn/memcached
+-   purl: pkg:rpm/redhat/memcached
+-   purl: pkg:rpm/centos/memcached
+-   purl: pkg:docker/library/memcached
+-   repology: memcached
+
+auto:
+-   git: https://github.com/memcached/memcached.git
+
+releases:
+-   releaseCycle: "1.6"
+    releaseDate: 2020-03-09
+    support: true
+    eol: false
+    latest: '1.6.22'
+    latestReleaseDate: 2023-10-09
+
+-   releaseCycle: "1.5"
+    releaseDate: 2017-07-21
+    support: false
+    eol: true
+    latest: '1.5.22'
+    latestReleaseDate: 2023-02-21
+---
+
+> memcached is a high-performance, distributed memory object caching system,
+> generic in nature, but originally intended for use in speeding up dynamic web
+> applications by alleviating database load.
+
+> You can think of it as a short-term memory for your applications.
```

Comment:
The production is missing a description of the versioning / release / support policy.

The most important one is the support policy. It's OK to "guess"or say it's not documented if the information is not available (see for example https://endoflife.date/apache-kafka).

Reply:
Ok I've updated it with what I guess from the releases and the project